lib/coverband.rb

```diff
@@ -28,6 +29,8 @@
 require 'coverband/integrations/background_middleware'
 require 'coverband/integrations/rack_server_check'
 
+Coverband::Adapters::RedisStore = Coverband::Adapters::HashRedisStore
```

Comment:
@danmayer Going to revert this. I just did this so i can test the HashRedisStore with the entire test suite. Thinking we want to keep both the RedisStore and the HashRedisStore around a while?

Reply:
yep, a ENV var to run the full suite with a different default store, or the like sounds like a great plan.